protected/humhub/modules/admin/views/setting/statistic.php

```diff
@@ -17,13 +17,15 @@
 <?= $form->errorSummary($model); ?>
 
 <div class="form-group">
-    <?= $form->field($model, 'trackingHtmlCode')->widget(CodeMirrorInputWidget::class); ?>
+    <?= $form->field($model, 'trackingHtmlCode')->widget(CodeMirrorInputWidget::className()); ?>
```

Comment:
This does not fix the issue causing the loading, not to mention is working backward with coding standards when using PHP 7.2+

Looking it over the issue seems to stem from the CodeMirrorInputWidget widget loading required scripts and CSS.